tests/functional/src/main/java/org/mule/functional/functional/FunctionalTestNotification.java

```diff
@@ -40,7 +40,15 @@
 
     static
     {
-        registerAction("event received", EVENT_RECEIVED);
+        //TODO(pablo.kraan): find a better way to manage this situation.
```

Comment:
This change was required when the classloader was created per test. Are you sure that is still required?